src/main/java/uk/gov/hmcts/darts/dailylist/controller/DailyListController.java

```diff
@@ -88,12 +89,9 @@ public ResponseEntity<PostDailyListResponse> dailylistsPost(PostDailyListRequest
     }
 
     @Override
-    public ResponseEntity<Void> dailylistsHousekeepingPost() {
```

Comment:
/dailylists/housekeeping will need removed from the yaml

Reply:
good point.